debug-cli/scripts/overwrite-key/recovery-script.sh

```diff
@@ -166,7 +166,7 @@ echo ""
 echo "[4/5] Updating key values using cdbg on leader broker ${LEADER_BROKER}..."
 
 # Build the cdbg command
-CDBG_CMD="/usr/local/camunda/bin/cdbg state update-key"
+CDBG_CMD="cdbg state update-key"
```

Comment:
❓ removing path? Ok?

Reply:
Yep it works. Looks like the camunda folder is part of `PATH`